.github/workflows/ci.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
         # Bookend python versions
-        python-version: ["3.10", "3.12"]
+        python-version: ["3.10", "3.12", "3.13"]
```

Comment:
shall we just stick to `all-but-dask` given what we've had to comment out?

Reply:
I'm in the progress of checking whether that's still necessary. If yes, then sure.

Reply:
okay, looks like this will still fail, but not for long. I don't think we can go with `all-but-dask`, though, since that's still trying to install `numba` / `sparse` / `numbagg`. Until that supports `python=3.13` we'll have to live with 3 more CI jobs, I guess.